src/main/java/pl/project13/maven/git/JGitProvider.java

```diff
@@ -233,8 +216,7 @@ protected void finalCleanUp() {
     try {
       DescribeResult describeResult = DescribeCommand
         .on(repository)
-        .withLoggerBridge(super.loggerBridge)
-        .setVerbose(super.verbose)
+        .withMojo(mojo)
```

Comment:
Verbose does what name prescribes - switches verbose logging on and off. Here it is removed because DescribeResult has no output - no logger - no verbose.